src/utils.ts

```diff
@@ -74,21 +74,14 @@ export function makeTmpDir() {
 }
 
 export function toWinDirFormat(dir: string = ''): string {
-  const results = /\/mnt\/([a-z])\//.exec(dir);
-  if (!results) {
-    return dir;
-  }
-
-  const driveLetter = results[1];
-  return dir.replace(`/mnt/${driveLetter}/`, `${driveLetter.toUpperCase()}:\\`)
-      .replace(/\//g, '\\');
+  return execFileSync('wslpath', [dir]).toString().trim();
 }
 
 export function getLocalAppDataPath(path: string): string {
-  const userRegExp = /\/mnt\/([a-z])\/Users\/([^\/:]+)\/AppData\//;
+  const userRegExp = /\/([a-z])\/Users\/([^\/:]+)\/AppData\//;
   const results = userRegExp.exec(path) || [];
 
-  return `/mnt/${results[1]}/Users/${results[2]}/AppData/Local`;
+  return toWinDirFormat(`C:\\${results[1]}\\Users\\${results[2]}\\AppData\\Local`);
```

Comment:
does the hardcoding of `C:` regress anything? AFAIK Windows makes it extraordinarily difficult to use a drive other than C: for this, but I haven't been up on latest windows config in quite some time

Reply:
That's completely a mistake - the drive letter comes is what `results[1]` holds.